susemanager-sync-data/additional_products.json

```diff
@@ -1,4 +1,91 @@
 [
+        {
+        "id" : -21,
```

Comment:
You should remove this section. We just added OES 2018 SP3 which has now `id : -21`

Reply:
Removed. 

We should have at least a wiki page, or a spreadsheet, with all of those values and the possibility to reserve entries so that when several people are working on adding new products at the same time (not that common but may happen), there are no clashes.